README.md

```diff
@@ -9,121 +9,34 @@ to events and their promotion.
 
 ## Installation
 
-mobilizon-reshare is distributed through Pypi. Use 
+`mobilizon_reshare` is distributed through Pypi and DockerHub. Use 
 
-```pip install mobilizon-reshare```
+```shell
+$ sudo pip install mobilizon-reshare
```

Comment:
remove sudo and leave permission handling to the user

Reply:
fixed, thanks